packages/fxa-auth-server/lib/senders/emails/layouts/fxa/index.ts

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { metaData } from '../../partials/metadata';
+
+export const render = (body: string, locAttributes: string) => {
+  return `
+<mjml>
+  <mj-head>
+    <mj-raw>
+      <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+      ${metaData}
+    </mj-raw>
+    <mj-title><%= subject %></mj-title>
+    ${locAttributes}
+  </mj-head>
+  <mj-body css-class="body-80">
+  <mj-include path="./lib/senders/emails/css/fxa/index.css" type="css" css-inline="inline" />
+  <mj-include path="./lib/senders/emails/css/global.css" type="css" css-inline="inline" />
+      <mj-raw>
+      <% if(preHeader){ %>
+        <div class="hidden-div">
+        <%= preHeader %>
+          &nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;
+          &nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;
+          &nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;
+          ‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;
+          ‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌&nbsp;‌‌‌
+        </div>
+        <% } %>
+      <% if(typeof(sync) == "undefined" || sync == null){ %>
+      <img
+             src="https://accounts-static.cdn.mozilla.net/product-icons/firefox-logo.png"
+             class="fxa-logo"
+            />
```

Comment:
Some funky indentation, maybe more like this?

```
<img
  src="https://accounts-static.cdn.mozilla.net/product-icons/firefox-logo.png"
  class="fxa-logo"
/>
```

That's how we write it for React components at least